lib/livebook_web/live/home_live.ex

```diff
@@ -170,14 +170,21 @@ defmodule LivebookWeb.HomeLive do
   end
 
   defp path_openable?(path, session_summaries) do
-    File.regular?(path) and not path_running?(path, session_summaries)
+    File.regular?(path) and not path_running?(path, session_summaries) and file_writable?(path)
   end
 
   defp path_running?(path, session_summaries) do
     running_paths = paths(session_summaries)
     path in running_paths
   end
 
+  defp file_writable?(path) do
+    case File.stat(path) do
+      {:ok, stat} -> stat.access in [:read_write, :write]
+      {:error, _} -> false
+    end
+  end
```

Comment:
Can we show a tooltip on the button? Something saying: "Can't open because file is read-only"?

Reply:
Something like this?

![image](https://user-images.githubusercontent.com/17034772/114725911-33a5c580-9d3d-11eb-9d64-56a52bbe6c23.png)

Reply:
Beautiful! 😍